_data/projects/RESTEasyCLI.yml

```diff
@@ -0,0 +1,12 @@
+name: RESTEasyCLI
+desc: Handy REST API client on your terminal
+site: https://github.com/rapidstack/RESTEasyCLI
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- cli
+- yml
+- REST
+upforgrabs:
+  name: help needed
```

Comment:
@suryasr007 Please change this to "help wanted" as your label link is for "help wanted" only.

Reply:
Thanks for the info. Done